README.md

```diff
@@ -76,7 +76,7 @@ The same works for `ingredients` as well
 # elements.yml
 - name: secrets
   ingredients:
-    - name: passwords
+    - role: passwords
```

Comment:
interesting. when has this change been made? is there a way to amend this?

Reply:
This was an old commit in the past: https://github.com/AlchemyCMS/alchemy-pg_search/commit/5be29b5296f49e4c17d8bb65160dac556f2ee53f